gengokit/httptransport/templates_test.go

```diff
@@ -201,7 +197,7 @@ func DecodeHTTPSumZeroRequest(_ context.Context, r *http.Request) (interface{},
 `
 	if got, want := str, desired; got != want {
```

Comment:
Thanks for the call out on table driven tests Ian, I've seen them elsewhere and used them in many tests within truss, but I didn't know their name.

Also, as far as where we found this `got, want` pattern, it is largely drawn from the code for `grpc-gateway`, where they make heavy use of it.](Thanks for the callout on table driven tests Ian, I've seen them elsewhere and used them in many tests within truss, but I didn't know their name.

Also, as far as where we found this `got, want` pattern, it is largely drawn from [the tests for `grpc-gateway`, where they make heavy use of it.](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/protoc-gen-grpc-gateway/descriptor/registry_test.go#L80)

For right now, I'm going to leave this test logic how it is. But going forward, I will be trying to shift to more table driven tests.